README.md

```diff
@@ -61,7 +63,7 @@ You can propose popular transformer rules to be included in the plugin by [sugge
 
 **How do I post articles to Instant Articles after plugin is installed?**
 
-In order to import your posts to your Instant Articles library on Facebook you need to either share them on your page or use the [Sharing Debugger](https://developers.intern.facebook.com/tools/debug/sharing/) to scrape them. After you have 10 articles imported, you will be able to submit them for review.
+In order to import your posts to your Instant Articles library on Facebook first you need to [Connect Your Site](https://developers.facebook.com/docs/instant-articles/guides/publishertools). Then either share your posts on your page or use the [Sharing Debugger](https://developers.intern.facebook.com/tools/debug/sharing/) to scrape them. After you have 10 articles imported, you will be able to submit them for review.
```

Comment:
~~first~~ you need to Connect Your Site **first**

Reply:
Done, thanks!